### DISCUSSION TITLE
繳交時waveform的內容為何

### DISCUSSION THREAD

**Discussion by BarryJu1999** [bol-edu/HLS-SOC-Discussions#110] [Q&A]
想請問助教在繳交waveform至github和壓縮檔時，是需要上傳.vcd檔案，
還是截圖說明內容，抑或是其他方式呢？

**Comment by Lanxas1018** [comment]
github上放截圖就好，說明內容可以在報告裡呈現。

**Comment by boledulab** [comment]
確定你截圖的waveform, 能說的出故事！
